doc/src/devdocs/inference.md

```diff
@@ -31,8 +31,7 @@ params = Core.Compiler.Params(typemax(UInt))  # parameter is the world age,
                                                         #   typemax(UInt) -> most recent
 sparams = Core.svec()      # this particular method doesn't have type-parameters
```

Comment:
```suggestion
sparams = Core.svec(Int64)      # this particular method has a type-parameter
```

Reply:
Running this code sample would previously have permanently corrupted the internal state of the system due to this